apps/vaporgui/LeftPanel.cpp

```diff
@@ -35,22 +34,35 @@ LeftPanel::LeftPanel(ControlExec *ce, MainForm *mf) : _ce(ce)
 
 void LeftPanel::Update()
 {
-    bool noDatasetLoaded = _ce->GetParams<GUIStateParams>()->GetOpenDataSetNames().empty();
-    if (noDatasetLoaded) return;
+    GUIStateParams *gsp = _ce->GetParams<GUIStateParams>();
+    // Enable the only import panel when no data is loaded
+    bool noDatasetLoaded = gsp->GetOpenDataSetNames().empty();
+    if (noDatasetLoaded) { 
+        setEnabled(true);
```

Comment:
MainForm has an explicit mechanism for controlling whether widgets are enabled based on data being loaded and LeftPanel is controlled by this mechanism. Please update the existing mechanism rather than fighting it here.

Reply:
This contradicts `QWidget::setEnabled(bool)`

Reply:
I don't understand the problem here.  We need to always enable the import tab, even when setEnabled(false) is issued.